consensus/src/network.rs

```diff
@@ -133,6 +133,17 @@ pub enum IncomingRpcRequest {
     RandGenRequest(IncomingRandGenRequest),
 }
 
+impl IncomingRpcRequest {
+    pub fn epoch(&self) -> Option<u64> {
+        match self {
+            IncomingRpcRequest::BatchRetrieval(req) => Some(req.req.epoch()),
+            IncomingRpcRequest::DAGRequest(req) => Some(req.req.epoch()),
+            IncomingRpcRequest::RandGenRequest(req) => Some(req.req.epoch()),
+            IncomingRpcRequest::BlockRetrieval(_) | IncomingRpcRequest::CommitRequest(_) => None,
```

Comment:
we should be able to parse epoch numbers from CommitRequest that's what actually log errors too

Reply:
Updated the PR.